games-engines/openscenegraph/openscenegraph-3.6.0.recipe

```diff
@@ -0,0 +1,198 @@
+	devel:libosgVolume$secondaryArchSuffix = $libVersionCompat
+	devel:libosgWidget$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	openscenegraph$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	mesa${secondaryArchSuffix}_devel
+#	devel:libavcodec$secondaryArchSuffix
+	devel:libcairo$secondaryArchSuffix
+	devel:libcurl$secondaryArchSuffix
+	devel:libEGL$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgdal$secondaryArchSuffix
+	devel:libgl$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libgif$secondaryArchSuffix
+#	devel:libgstreamer_1.0$secondaryArchSuffix
+	devel:libhalf$secondaryArchSuffix
+	devel:libIlmImf$secondaryArchSuffix
+	devel:libintl$secondaryArchSuffix
+	devel:libjasper$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+#	devel:libOpenCOLLADABaseUtils$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+	devel:libpoppler${secondaryArchSuffix}_glib
+	devel:libsdl$secondaryArchSuffix
+	devel:libtiff$secondaryArchSuffix
+#	devel:libTKGeomBase$secondaryArchSuffix #opencascade
+	devel:libxml2$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+defineDebugInfoPackage openscenegraph$secondaryArchSuffix \
+	$libDir/libOpenThreads.so.$libOpenThreadsVersion \
+	$libDir/libosg.so.$libVersion \
+	$libDir/libosgAnimation.so.$libVersion \
+	$libDir/libosgDB.so.$libVersion \
+	$libDir/libosgFX.so.$libVersion \
+	$libDir/libosgGA.so.$libVersion \
+	$libDir/libosgManipulator.so.$libVersion \
+	$libDir/libosgParticle.so.$libVersion \
+	$libDir/libosgPresentation.so.$libVersion \
+	$libDir/libosgShadow.so.$libVersion \
+	$libDir/libosgSim.so.$libVersion \
+	$libDir/libosgTerrain.so.$libVersion \
+	$libDir/libosgText.so.$libVersion \
+	$libDir/libosgUI.so.$libVersion \
+	$libDir/libosgUtil.so.$libVersion \
+	$libDir/libosgViewer.so.$libVersion \
+	$libDir/libosgVolume.so.$libVersion \
+	$libDir/libosgWidget.so.$libVersion
+
+BUILD()
+{
+	mkdir -p build && cd build
+	cmake .. \
+		-DCMAKE_INSTALL_PREFIX=$prefix \
+		-DBUILD_OSG_APPLICATIONS=OFF \
+		-DOSG_WINDOWING_SYSTEM=None \
+		$cmakeDirArgs
+	make $jobArgs
+}
+
+INSTALL()
+{
+	cd build
+	make install
+
+	mkdir -p $includeDir
+	mv $prefix/include/* $includeDir/
+	rm -rf $prefix/include
+
+	# Secondary-arch libdir cleanup
+	mkdir -p $prefix/lib-temp
+	mv $prefix/lib/* $prefix/lib-temp/
+	mkdir -p $libDir
+	mv $prefix/lib-temp/* $libDir
+	rm -rf $prefix/lib-temp
+
+	prepareInstalledDevelLibs libOpenThreads libosg libosgAnimation libosgDB \
+		libosgFX libosgGA libosgManipulator libosgParticle libosgPresentation \
+		libosgShadow libosgSim libosgTerrain libosgText libosgUI libosgUtil \
+		libosgViewer libosgVolume libosgWidget
+
+	fixPkgconfig
+
+	packageEntries devel \
+		$developDir
+}
+
+TEST()
+{
+	cd build
+	make test
```

Comment:
@extrowerk, are you sure this worked for `3.6.0`?
Because with `3.6.3` (see #3054) there is no such `test` target:
```
haikuporter --test openscenegraph_x86
...
Testing ...
make: *** No rule to make target 'test'.  Stop.
Warning: Command '['bash', '-c', '. /wrapper-script']' returned non-zero exit status 2
Error: Test has failed - stopping.
```